pages/api/lockerbox/create.ts

```diff
@@ -0,0 +1,28 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+import { Role, User } from '../../../types/types'
+import { v4 as uuidv4 } from 'uuid'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { lockerId, locker_count } = req.body
+
+      let lockerBoxes = Array.from({ length: locker_count }, (_, i) => {
+        let obj = { lockerId: lockerId, label: i + 1, status: 'empty' }
+        return obj
+      })
```

Comment:
This can be reduced to shorthand I believe
```suggestion
      const lockerBoxes = Array.from({ length: locker_count }, (_, index) => ({ lockerId, label: index + 1, status: 'empty' }))
```